src/coreclr/src/pal/src/arch/arm64/activationhandlerwrapper.S

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "unixasmmacros.inc"
+#include "asmconstants.h"
+
+// Offset of the return address from the ActivationHandler in the ActivationHandlerWrapper
+.globl C_FUNC(ActivationHandlerReturnOffset)
```

Comment:
We will need to verify that stack unwinding in LLDB is correct when the ActivatonHandlerWrapper is on the stack once we can execute managed code.

Reply:
LGTM
```
* thread #1, queue = 'com.apple.main-thread', stop reason = breakpoint 1.1
    frame #0: 0x00000001010b6df8 libcoreclr.dylib`::ActivationHandler(context=0x000000016fdfd010) at machexception.cpp:1423:9
   1420 void
   1421 ActivationHandler(CONTEXT* context)
   1422 {
-> 1423     if (g_activationFunction != NULL)
   1424     {
   1425 #if defined(HOST_OSX) && defined(HOST_ARM64)
   1426         auto jitWriteEnableHolder = PAL_JITWriteEnable(false);
Target 0: (corerun) stopped.
(lldb) bt 20
* thread #1, queue = 'com.apple.main-thread', stop reason = breakpoint 1.1
  * frame #0: 0x00000001010b6df8 libcoreclr.dylib`::ActivationHandler(context=0x000000016fdfd010) at machexception.cpp:1423:9
    frame #1: 0x00000001010b4400 libcoreclr.dylib`ActivationHandlerWrapper at activationhandlerwrapper.S:16
    frame #2: 0x000000028182e5a0
    frame #3: 0x000000028182e534
    frame #4: 0x000000028182cc2c
    frame #5: 0x0000000281829c24
    frame #6: 0x00000002818287c4
    frame #7: 0x000000028182674c
    frame #8: 0x00000002818250b0
    frame #9: 0x0000000281824ad4
    frame #10: 0x000000028139e088
    frame #11: 0x000000028139c8cc
    frame #12: 0x0000000281399a24
    frame #13: 0x000000028139607c
    frame #14: 0x000000028138c63c
    frame #15: 0x0000000101641a4c libcoreclr.dylib`CallDescrWorkerInternal at calldescrworkerarm64.S:71
    frame #16: 0x0000000101345aa8 libcoreclr.dylib`CallDescrWorkerWithHandler(pCallDescrData=0x000000016fdfe950, fCriticalCall=NO) at callhelpers.cpp:71:5
    frame #17: 0x00000001013465fc libcoreclr.dylib`MethodDescCallSite::CallTargetWorker(this=0x000000016fdfeb38, pArguments=0x000000016fdfeac8, pReturnValue=0x000000016fdfea28, cbReturnValue=8) at callhelpers.cpp:547:9
    frame #18: 0x0000000101100ba4 libcoreclr.dylib`MethodDescCallSite::Call_RetArgSlot(this=0x000000016fdfeb38, pArguments=0x000000016fdfeac8) at callhelpers.h:458:9
    frame #19: 0x000000010110092c libcoreclr.dylib`RunMainInternal(pParam=0x000000016fdfed40) at assembly.cpp:1468:48
```